amqp/method_framing.py

```diff
@@ -94,11 +108,17 @@ def frame_writer(connection, transport,
     # memoryview first supported in Python 2.7
     # Initial support was very shaky, so could be we have to
     # check for a bugfix release.
```

Comment:
This comment is probably out of place or even obsolete.
Do you mind issuing a new PR to remove it?

Reply:
Good catch, I'll put something up.

Reply:
@thedrow https://github.com/celery/py-amqp/pull/329. I think the best option is to just remove it since it seems likely to not be an issue at this point.